src/Mailer/Mailer.php

```diff
@@ -224,7 +225,7 @@ private function sendMailFromTemplate(
         $email->htmlTemplate('emails/' . $templateName . '.twig');
         $email->context(array_merge($parameters, [
             'fullRegistrationLink' => $this->settings->getFullUrlLink(),
-            'eventImageExists' => is_file(__DIR__ . '/../../public/' . $event->logoUrl),
+            'eventImageExists' => is_file(__DIR__ . '/../../public/' . Event::getFullLogoUrl($event->logoUrl)),
```

Comment:
Není potřeba volat staticky, když máme k dispozici už konkrétní objekt, kde to je